api/v1/MEL.php

```diff
@@ -122,6 +122,13 @@ private function SetupMELLayer(string $melLayerName, array $config): int
             "SELECT layer_id, layer_raster FROM layer WHERE layer_name=?",
             array($layerName)
         );
+
+        $game = new Game();
+        $globalConfig = $game->GetGameConfigValues();
+        $layerMeta = current(array_filter($globalConfig['meta'], function ($meta) use ($layerName) {
+            return strcasecmp($meta['layer_name'], $layerName) === 0;
+        }));
+        $layerName = $layerMeta['layer_name'] ?? $layerName;
```

Comment:
@hwarmelink Interessant. Ik loop tegen een issue aan met case sensitive files . 

nl. deze error :
Could not extract region from image: unable to open image '/srv/app/raster/1/mel_Protection_Industrial_Trawl.tif

Check de "url' waarde in de database. Dat is idd mel_Protection_Industrial_Trawl met een capital Trawl :
`
MariaDB [msp_session_1]> select layer_raster from layer where layer_name like 'mel_Protection_Industrial_Trawl'
    -> ;
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| layer_raster                                                                                                                                                                                                                                                                                  |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| {"layer_raster_material":"RasterMELCustomizable","layer_raster_pattern":"Default","layer_raster_minimum_value_cutoff":0.05,"layer_raster_color_interpolation":0,"layer_raster_filter_mode":1,"url":"mel_Protection_Industrial_Trawl.tif","boundingbox":[[3463000,3117000],[4313000,4257000]]} |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
`

Maar op bestandsniveau is het lowercase trawel:
`
/srv/app # ls /srv/app/raster/1/mel_Protection_
mel_Protection_bottom_trawl.tif      mel_Protection_industrial_trawl.tif  mel_Protection_nets.tif
`

En dan komt dan weer overeen met de North_Sea_basic_1.json:
`
      {
        "layer_name": "mel_Protection_industrial_trawl",
        "layer_geotype": "raster",
`

In MEL.php doen we dit:
`
private function SetupMELLayer(string $melLayerName, array $config): int
    {
        $layerName = "mel_" . str_replace(" ", "_", $melLayerName);
        $data = $this->getDatabase()->query(
            "SELECT layer_id, layer_raster FROM layer WHERE layer_name=?",
            array($layerName)
        );
        $rasterProperties = array(
            "url" => "$layerName.tif",
            "boundingbox" => array(
                array($config["x_min"], $config["y_min"]), array($config["x_max"], $config["y_max"])
            )
        );
`
Dus ik vermoed dat hier de "url" waarde wordt aangemaakt die uiteindelijke in layer_raster in de tabel layer terecht komt voor mel layers.

Eigenlijk moeten we de naam in de xml corrigeren . 

Maar een fix kan zijn om in ieder geval de "url" te laten matchen met de naam van de layer. Dus de mismatch tussen layer_name en de url in layer_raster moet weg: 
`
MariaDB [msp_session_1]> select layer_name, layer_raster from layer where layer_name = 'mel_Protection_Industrial_trawl';
+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| layer_name                      | layer_raster                                                                                                                                                                                                                                                                                  |
+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| mel_Protection_industrial_trawl | {"layer_raster_material":"RasterMELCustomizable","layer_raster_pattern":"Default","layer_raster_minimum_value_cutoff":0.05,"layer_raster_color_interpolation":0,"layer_raster_filter_mode":1,"url":"mel_Protection_Industrial_Trawl.tif","boundingbox":[[3463000,3117000],[4313000,4257000]]} |
+---------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
`

En dan is dit volgens mij de fix om de naam
`$layerName = "mel_" . str_replace(" ", "_", $melLayerName);`
op te zoeken in de config file in meta maar dan case insensitive , en dan de layer_name van die meta data te pakken .

Dus zo:
`
        $game = new Game();
        $globalConfig = $game->GetGameConfigValues();
        $layerMeta = array_filter($globalConfig['meta'], function ($meta) use ($layerName) {
            return strcasecmp($meta['layer_name'], $layerName) === 0;
        });
        $layerName = $layerMeta['layer_name'] ?? $layerName;
        $rasterProperties = array(
            "url" => "$layerName.tif",
            "boundingbox" => array(
                array($config["x_min"], $config["y_min"]), array($config["x_max"], $config["y_max"])
            )
        );
`

Ik heb het getest en het werkt nu . En het is toegevoegd aan de PR

Reply:
```suggestion
        // take the config's layer name since the case of the characters can be different from MEL's. 
        $layerName = $layerMeta['layer_name'] ?? $layerName;
```